sphinx_gallery/backreferences.py

```diff
@@ -165,9 +165,9 @@ def scan_used_functions(example_file, gallery_conf):
 
 .. only:: html
 
-    .. figure:: /{thumbnail}
+ .. figure:: /{thumbnail}
```

Comment:
FYI these indentation changes are useful because they prevent whitespace from being added in the HTML. Instead of having:
```
   .. figure::
```
this comes out with no leading whitespace as:
```
.. figure::
```
This is better RST, and also necessary if we want to replace the header with `.. rubric::` (which is what I'm using over in MNE to get methods and attributes linked using backrefs)